packages/hoppscotch-sh-admin/src/helpers/auth.ts

```diff
@@ -386,12 +240,12 @@ export const auth = {
     }
   },
 
-  async signOutUser(reloadWindow = false) {
+  signOutUser: async (reloadWindow = false) => {
     await signOut(reloadWindow);
   },
```

Comment:
Could you see if it can be refactored to return the promise straightaway instead of awaiting?